docs/setup/schema.md

```diff
@@ -81,3 +103,41 @@ or missing may cause the service to crash. If you are the running the DICOM Adap
 Kubernetes/Helm, you may see the `CrashLoopBack` error.  To review the validation errors, simply
 run `kubectl logs <name-of-dicom-adapter-pod>`.
 
+## Logging
+
+DICOM Adapter, by default, write all logs to console.  If DICOM Adapter is running inside a Docker container, additional configuration may be required to limit the size to prevent filling up storage space.  Please refer to [Docker](https://docs.docker.com/config/containers/logging/configure/) for additional information.
+
+
+### Log Levels
+Log level may be adjusted per module basis.  For example, given the following log entries:
+
+```
+10:31:03 info: Nvidia.Clara.DicomAdapter.Server.Processors.AeTitleJobProcessor[0]
+      Initializing AE Title DicomWebTest with processor Nvidia.Clara.DicomAdapter.Server.Processors.AeTitleJobProcessor, Nvidia.Clara.DicomAdapter
+10:31:03 info: Nvidia.Clara.DicomAdapter.Server.Processors.AeTitleJobProcessor[0]
+      AE Title DicomWebTest Processor Setting: timeout=5s
+```
+
+By default, the `Nvidia` namespace is set to log all `Information` level logs.  If additional information is required to debug the **AE Title Job Processor module** or to turn down the noise, simply add a new entry under the `LogLevel` section of the configuration file to adjust it:
+
+```
+ "Logging": {
+    "LogLevel": {
+      "Nvidia": "Information",
+      "Nvidia.Clara.DicomAdapter.Server.Processors.AeTitleJobProcessor": "Debug",
```

Comment:
Is there a comprehensive list of processors available in the docs that I can refer to which I may choose to add or modify here?
e.g. is there a `Nvidia.Clara.DicomAdapter.Server.Processors.SourceJobProcessor`?

Reply:
`AeTitleJobProcessor` is the only Job Processor we have.